### PR TITLE
Scopes - UI improvement

### PR DESCRIPTION
**Pull request by ulteq**
| Before  | After |
| ------------- | ------------- |
| before1wk9i | after7mjid |

### CODE REVIEW

**Comment by PabstMirror** [comment]
Stringtables for the letters?

**Comment by ulteq** [comment]
Right!

**Comment by bux** [comment]
That's printed on the RL turrets?

**Comment by ulteq** [comment]
Yes, it's very common that some kind of direction indicator is printed on the turret itself. Together with a little arrow.

Common letterings are, `UP`, `RIGHT`, `LEFT`, `R`, `L`.

**Comment by ulteq** [comment]
But I don't know if we should translate them. At first I thought it's obvious that we should. But I'm not sure anymore.

**Comment by Laid3acK** [comment]
French military SCROME scopes have an `H` (up), `B` (down), `G` (left) and `D` (right) on the turrets:
http://www.scrome.com/assets/templates/flexibility/pdf/Scrome_Marksman_Scope_LTE_Datasheet_GB.pdf

Russian scopes have also their own Cyrillic indications.

**Comment by ulteq** [comment]
I will add string table entries for all four directions.

**Comment by ulteq** [comment]
Should we optionally offer the old "signed" version?

File: addons/scopes/stringtable.xml [hunk 1]
```diff
@@ -365,5 +365,25 @@
             <Chinesesimp>此模块可为高倍率瞄准镜新增归零风偏，距离用的调整纽。</Chinesesimp>
             <Chinese>此模塊可為高倍率瞄準鏡新增歸零風偏，距離用的調整紐。</Chinese>
         </Key>
+        <Key ID="STR_ACE_Scopes_DisplayAdjustmentUp">
```
Comment (bux) [inline addons/scopes/stringtable.xml]: There seems something wrong with the translations.
https://github.com/acemod/ACE3/pull/5702#issuecomment-340248564

**Pull request merged**